iterations.r

```diff
@@ -40,30 +42,43 @@ stations_metadata_df <-
   stations_metadata %>% 
   transform_metadata_to_df(.)
 
+stations_metadata_df
 
 #### 3: Testing metadata
 source("functions/data_tests.r")
 test_stations_metadata(stations_metadata_df)
 
+### 4: test
+GQL(
+  vol_qry(
+    id=stations_metadata_df$id[1], 
+    from=to_iso8601(stations_metadata_df$latestData[1],-4),
+    to=to_iso8601(stations_metadata_df$latestData[1],0)
+  ),
+  .url = configs$vegvesen_url
+)
 
 ### 5: Final volume query: 
 
 source("gql-queries/vol_qry.r")
 
-stations_metadata_df %>% 
-  filter(latestData > Sys.Date() - days(7)) %>% 
-  sample_n(1) %$% 
+stations_metadata_df %>%
+  filter(latestData > Sys.Date() - days(7)) %>%
+  sample_n(1) %>%
+  {station_name <<- .$name[1]; .} %$%
   vol_qry(
     id = id,
     from = to_iso8601(latestData, -4),
     to = to_iso8601(latestData, 0)
   ) %>% 
-  GQL(., .url = configs$vegvesen_url) %>%
+  GQL(.url = configs$vegvesen_url) %>%
   transform_volumes() %>% 
   ggplot(aes(x=from, y=volume)) + 
   geom_line() + 
-  theme_classic()
-
-
+  theme_classic() +
+  labs(title = "Traffic Volume over Time",
+       subtitle = paste("Traffic Station:", station_name),
+       x = "Time",
+       y = "Volume")
```

Comment:
Really good plot. I like that you added a dynamic title showing which station it is. Also, it is good that you made the axis labels nicer, as the first letter in the default titles is not uppercase.